packages/@react-aria/table/src/useTable.ts

```diff
@@ -21,6 +21,7 @@ import {useId} from '@react-aria/utils';
 import {useMemo} from 'react';
 
 interface TableProps<T> extends GridProps {
+  /** The layout object for the table. Computes what content is visible and how to position and style them. */
   layout?: Layout<Node<T>>
 }
 
```

Comment:
Missing a doc comment for the function below? Also a bunch of the other hooks in this PR.

Reply:
Ah good catch, will add